apis/addonconfigs/config/crd/bases/cpi.tanzu.vmware.com_vspherecpiconfigs.yaml

```diff
@@ -57,7 +57,6 @@ spec:
                     description: The datacenter in which VMs are created/located
                     type: string
                   insecure:
-                    default: false
```

Comment:
Why remove the default? 
won't the added code set this to true if tlsthumbprint is provided.

Reply:
I see, so now thumbprint must be provided. as longs thumbprintTLS = "" is the default then it should be fine.